public/scripts/controllers/user.js

```diff
@@ -26,8 +27,28 @@ function UserController($modal, user) {
     });
   };
 
+  this.editProfile = function() {
+    this.isEditing = true;
```

Comment:
Actually found out `this` maintains its reference in a single nesting. `this.isEditing` performs as expected. Were it inside another function, then the alias is required.